docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
       context: .
       dockerfile: Dockerfile
     ports:
-      - "6060:8080"
+      - "8080:8080"
```

Comment:
this causes problems in my setup, but i'll fix it through env variables when i finally get around to working on the dockers again